### PR TITLE
update(config/jobs): add prow jobs for minikube drivers

### PR DESCRIPTION
**Pull request by alacuku**
This PR adds two new jobs to handle the minikube drivers.

### CODE REVIEW

File: config/jobs/build-drivers/rebuild-all-minikube.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        resources:
+          requests:
+            cpu: 1500m #m5large is 2vpcu and 8gb ram so this 75% of a node
+            memory: 6Gi
+      nodeSelector:
+        Archtype: "x86"
+periodics:
+  - name: rebuild-all-drivers-minikube-periodic
+    decorate: true
+    cron: "0 03 * * 0"
```
Comment (alacuku) [inline config/jobs/build-drivers/rebuild-all-minikube.yaml:36]: Not sure about the cron settings.

**Comment by alacuku** [comment]
Before merging this PR we need:

- a new release of driverkit containing falcosecurity/driverkit#189;
- update the driverkit version used by the test infra to build the kmod and probe.

**Comment by leogr** [comment]
Hey @alacuku 

> * update the driverkit version used by the test infra to build the kmod and probe.

Could you make a PR for that?

**Comment by alacuku** [comment]
Hey Leo,
> Could you make a PR for that?

I'm waiting for driverkit to be released. Once released I'll bump the driverkit version!

**Comment by leogr** [comment]
> Hey Leo,
> 
> > Could you make a PR for that?
> 
> I'm waiting for driverkit to be released. Once released I'll bump the driverkit version!

cc @FedeDP @dwindsor

**Comment by FedeDP** [comment]
Hey leo, i will hopefully release a new driverkit version soon enough :)

**Comment by FedeDP** [comment]
Driverkit 0.9.6 is now released :) https://github.com/falcosecurity/driverkit/releases/tag/v0.9.6

**Comment by alacuku** [comment]
> Hey @alacuku
> 
> > * update the driverkit version used by the test infra to build the kmod and probe.
> 
> Could you make a PR for that?

Hi @leogr, 

I opened the PR to bump the version of driverkit here: #806. All the required pieces are in place now so we can proceed and merge these two PRs. 

cc @FedeDP

File: config/jobs/build-drivers/rebuild-all-minikube.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      - ^master$
+    always_run: false
+    spec:
+      serviceAccountName: driver-kit
+      containers:
+      - command:
+        - /workspace/build-drivers.sh
+        - minikube
+        env:
+        - name: AWS_REGIONe
```
Comment (FedeDP) [inline config/jobs/build-drivers/rebuild-all-minikube.yaml:17]:
```suggestion
        - name: AWS_REGION
```

**Comment by leogr** [comment]
Since #806 got merged we can unblock this
/unhold

**Pull request merged**